ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -54,6 +54,26 @@ RSkComponentTextInput::RSkComponentTextInput(const ShadowView &shadowView)
   sem_init(&jsUpdateSem,0,0);
   isKeyRepeateOn=false;
 }
+SkPaint RSkComponentTextInput::convertTextColor ( SharedColor textColor ) {
+  SkPaint paint;
+  float ratio = 255.9999;
+  auto color = colorComponentsFromColor(textColor);
+  paint.setColor(SkColorSetARGB( color.alpha * ratio,
+                                     color.red * ratio,
+                                     color.green * ratio,
+                                     color.blue * ratio));
+  paint.setAntiAlias(true);
+  return paint;
+}
+
+
+void RSkComponentTextInput::drawUnderline(SkCanvas *canvas,Rect frame,SharedColor underlineColorAndroid){
+  SkPaint paint = convertTextColor(underlineColorAndroid);
+  paint.setAntiAlias(true);
+  paint.setStyle(SkPaint::kStroke_Style);
+  paint.setStrokeWidth(1);
+  canvas->drawLine(frame.origin.x,frame.origin.y+frame.size.height-3,frame.origin.x+frame.size.width,frame.origin.y+frame.size.height-3, paint);
```

Comment:
than deferecing the frame.origin and frame.size multiple times, can it be stored in variable and used.

Reply:
is textAlignVertical configuration not effecting this line positioning ?

Reply:
textAlignVertical --> there is no impact on the textinput, I verified on android(as a style Props). Update same in jira.

Reply:
than deferecing the frame.origin and frame.size multiple times, can it be stored in variable and used. --> discussed Offline